README.md

```diff
@@ -8,158 +8,115 @@ Any idea is :heart:, let discuss about it on [![Join the chat at https://gitter.
 ## Result
 
 <!-- Result from here -->
-Last update: 2018-05-12
+Last update: 2018-06-08
 ```
-OS: Linux (version: 4.16.7-200.fc27.x86_64, arch: x86_64)
-CPU Cores: 4
+OS: Linux (version: 4.16.11-100.fc26.x86_64, arch: x86_64)
+CPU Cores: 8
 ```
 
 ### Ranking (Framework)
```

Comment:
Should the req/s and maybe throutput be listed with these as well so people can see the differences in the ranking, not just the positional ranking?  Perhaps it should be a markdown table too?